components/lib/crash/src/test/java/mozilla/components/lib/crash/handler/CrashHandlerServiceTest.kt

```diff
@@ -69,6 +70,7 @@ class CrashHandlerServiceTest {
         doNothing().`when`(service).kill()
         service.onHandleIntent(intent)
 
+        sleep(100)
```

Comment:
Do we still need this one?

Reply:
Yes, I don't think I can avoid this one.

Reply:
Let's try to find an alternative, as this could cause a lot of flakiness in the tests. Later in the day I will take a closer look

Reply:
Sorry, I don't want to be picky, I just want to find a better alternative :)

Reply:
I understand.  I don't like it as well.  The test might have to be changed to not use the sleep.  The `onHandleIntent` API can't be modified to return the job.

Reply:
Created issue to track this https://github.com/mozilla-mobile/android-components/issues/5480

Reply:
@rocketsroger @Amejia481 let's not land the test with the `sleep`, please. :) Here's an alternative version of the test and change that should work reliably: https://github.com/csadilek/android-components/commit/fd57536ea59ab4250c8ea88e8618e798b159628a

Reply:
Thanks!

Reply:
Thanks! I've applied your changes to this PR.  I'll close the created issue.